content/teams/Tstar/index.md

```diff
@@ -0,0 +1,146 @@
+---
+type: "team"                   
+title: Tstar
+name: "Tstar"
+github-link: "https://github.com/Neelkukreti/CodeCampSubmission.git"
+description: "hello I am Bot Developer"
+Team-Quote: "Making ideas a reality"
+Team_Tech_Stack: "VR"
+Application_Type: "Mobile app"
+No_Of_Member: "2"
+---
+
+Hey ! We are Team Tstar.
+
+---
+
+## Project's Overview
+
+_**Theme-Name**_ : Empowering Differently Abled
+
+_**Application-Type**_ :   Mobile Application 
+
+_**Idea-Description**_ :   
+
+Solution focuses on tackling the mental health issues of people. Mentally disabled people often find it hard for them to get diagnosed or find treatment, this app seeks to solve that.
+It helps make the solution more accessible for everyone , using just a mobile phone and a 200Rs headset anyone can take care of their own mental health.
+According to recent surveys there is 1 therapist for every 80000 people in India, so therapy for everyone is not accessible, our VR therapist feature solves that by bringing a therapist you can talk to in your phone and through VR for the best immersion, this is achieved by using Azure’s speech to text engine in unity3D.
+Besides this , other mental health issues tackled by the solution are , anger issues, anxiety , various kinds of phobias, depression , demotivation , stress and to bundle it all up a guided meditation experience.
+Main tools used in this app are , unity3D engine , Google’s VR SDK, Android SDK, Azure, firebase (to be implemented).
+The solution’s USP is there is no such product in the market, its ease of use (just wear a headset), accessibility (works on the cheapest phones) , headset is readily available and is cheap , the app is free.
+There is immense potential for scaling this app, it can be used as a tool by psychiatrists to treat patients, patients can use it to take care of themselves, most features are available offline hence no server side setup is required, there is no such app that does VR mental health care (closest competitions are audio only solutions).
+This app can change the mental health care landscape of india by leveraging the VR tech and making it more accessible, potential is endless.
+
+Demo - https://youtu.be/Ax7dE_bSAsU
+
+More details - https://drive.google.com/file/d/1JiIjQ2llw0X2mn_BYni1FgbUc0H-Ec12/view?usp=sharing
+
+_**Tech-Stack-Used**_ :   Unity Game Engine C#
+
+_**GitHub-Link**_ :   https://github.com/Neelkukreti/CodeCampSubmission.git 
+
+_**Hosted-Link**_ :    https://drive.google.com/file/d/1OmgyR80H_UFbu2APdmX_GBf-2ok84uYt/view?usp=sharing
```

Comment:
Do not Give hosted link Now. You can't submit your project prior to the hackathon. it will be submitted on the submission day in the submission Pull Request.